taskcluster/ffios_taskgraph/release_promotion.py

```diff
@@ -0,0 +1,201 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Note: I didn't read this file. I know it's usually a copy-paste from another repo. Let me know if you want me to review specific parts of this file.